src/babylon.scene.ts

```diff
@@ -689,21 +696,25 @@ module BABYLON {
         /**
         * Gets or sets the fog color to use
         * @see http://doc.babylonjs.com/babylon101/environment#fog
+        * Default is Color3(0.2, 0.2, 0.3)

```

Comment:
Because the Doc will concatenate lines, can you add parenthesis around the default value line?